GameStore.Infrastructure/Persistence/Configurations/OrderConfiguration.cs

```diff
@@ -0,0 +1,39 @@
+﻿using GameStore.Domain.Entities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace GameStore.Infrastructure.Persistence.Configurations;
+
+public class OrderConfiguration : IEntityTypeConfiguration<Order>
+{
+    public void Configure(EntityTypeBuilder<Order> builder)
+    {
+        builder.Property(o => o.FirstName)
+               .IsRequired()
+               .HasMaxLength(20);
+
+        builder.Property(o => o.LastName)
+               .IsRequired()
+               .HasMaxLength(20);
+
+        builder.Property(o => o.Phone)
+               .HasMaxLength(15)
+               .IsRequired();
```

Comment:
```suggestion
               .IsRequired()
               .HasMaxLength(15);
```

Just for consistency